### PR TITLE
[EA Forum only] Community posts section

### PR DESCRIPTION
**Pull request by s-cheng**
As discussed [here](https://forum.effectivealtruism.org/posts/wvBfYnNeRvfEXvezP/moving-community-discussion-to-a-separate-tab-a-test-we), we are testing out hiding "Community" posts from the Frontpage section by default, and adding a new "Community posts" section underneath.

I also sentence-cased all the sections on the EAF home page.

Screen Shot 2023-02-07 at 11 53 06 AM

This also renames the ["Community" page](https://forum.effectivealtruism.org/community) to "Connect", to reduce confusion between that and the "Community" topic page. In the future we'll revisit what to call the "Connect" page.

-----
TODO when deploying this:
1. Update default "Community" tag filter setting in the db settings to be "Hidden"
2. Set "Community" tag filter setting to "Hidden" for all users who have edited their filter settings, via the script in this PR

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203922168114369) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx
Comment (s-cheng) [inline packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx:22]: This is pulled out in https://github.com/ForumMagnum/ForumMagnum/pull/6617 so I'll replace it in that PR

**Comment by agnesstenlund** [comment]
> Update default "Community" tag filter setting in the db settings to be "Hidden"

This may have changed since we last discussed it, but I was of the impression only `Community opinion` would get hidden by default? And `Community` would still just show as reduced

> I also sentence-cased all the sections on the EAF home page.

❤️

**Comment by imbenclifford** [comment]
Yeah we changed this as deciding what should count as "Community opinion" or not was tricky. Sorry for not making that clearer before.

**Review by Will-Howard — COMMENTED** [review]
> I also sentence-cased all the sections on the EAF home page.

🥇 

LGTM. It would be best to wait until I've merged this [PR](https://github.com/ForumMagnum/ForumMagnum/pull/6652) and the subforum -> core topics renaming before merging this. But if that's not done by the end of tomorrow I think it's fine to go ahead with it

**Review by Will-Howard — APPROVED** [review]
Sorry, meant to approve this yesterday

**Pull request merged**
